sql-common/client.cc

```diff
@@ -1374,6 +1374,24 @@ bool cli_advanced_command(MYSQL *mysql, enum enum_server_command command,
   if ((command != COM_QUIT) && mysql->reconnect && !vio_is_connected(net->vio))
     net->error = NET_ERROR_SOCKET_UNUSABLE;
 
+  /*
+    If the connection becomes not alive after the above check,
```

Comment:
Can we use `dead` or `disconnected` instead of `not alive`?
```suggestion
    If the connection becomes dead after the above check,
```

Reply:
done